vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -19,15 +19,22 @@ public final class TestValidatorScript extends TestValidator {
   private String outputMetrics;
 
   @Override
-  public Validator createValidator() {
+  public Validator createValidator(String outputDirectory, String id) {
     try {
-      final var tempDir = Files.createTempDirectory("vidarr-test-script");
-      final var calculateScript = tempDir.resolve("calculate");
-      final var calculateDir = tempDir.resolve("output");
-      Files.createDirectories(calculateDir);
-      Files.copy(Path.of(metricsCalculate), calculateScript);
+      // If output directory provided, use that else we create temporary directory /tmp
```

Comment:
Technically, we don't create /tmp, and where the temporary directory vidarr-test-script goes is OS specific: https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.lang.String-java.nio.file.attribute.FileAttribute...-

But the assumption with all our software is that it gets run on a unix-like so only a minor reword is needed here